node/engine/messageservice/p2p-message-service/service.go

```diff
@@ -323,44 +326,10 @@ func (ms *P2PMessageService) addBootPeers(peers []string) {
 		peer, err := peer.AddrInfoFromP2pAddr(addr)
 		ms.checkError(err)
 
-		err = ms.p2pHost.Connect(context.Background(), *peer) // Adds peerInfo to Peerstore
+		err = ms.p2pHost.Connect(context.Background(), *peer) // Adds peerInfo to local Peerstore
 		ms.checkError(err)
 		ms.logger.Debug().Msgf("connected to boot peer: %v", p)
 
 		ms.sendPeerInfo(peer.ID, true)
 	}
 }
-
-func (ms *P2PMessageService) discoverPeers(ctx context.Context, topic string) {
```

Comment:
We may want to modify and add this method back at some point to augment the libp2p connection notifications for faster, more robust peer discovery. However, the libp2p connection notifications replace this method for now.